web/src/pages/Sharedpage.tsx

```diff
@@ -0,0 +1,51 @@
+import { useNavigate } from "react-router";
+import JuiceSlide from "../component/main-shared/JuiceSlide.tsx";
+
+export default function Sharedpage() {
+  const navigate = useNavigate();
+
+  const toGoNext = () => {
```

Comment:
화살표 함수는 반환값이 1개뿐일때 중괄호를 생략할 수 있어
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Functions/Arrow_functions
```suggestion
const toGoNext = () => navigate("/step");
```

Reply:
중괄호 쓴 함수가 보기에 익숙해서 계속 썼었는데, 생략하는게 정배라면 이제부터 중괄호 생략한 모습에 익숙해져 보겠습니다~!!
이 파일 뿐만 아니라 다른 파일도 이렇게 되어 있는 부분 수정 완료 했습니다!